html.md

```diff
@@ -18,28 +18,29 @@ Most tags also have a matching closing tag that is written **almost** exactly th
 </tagname>
 ```
 
-Inside of the closing and opening tags, we can place content such as text. The full group of opening tag, content and a closing tag is what is known as an _element_.
+Inside of the closing and opening tags, we can place content such as text.
 
 ```html
 <tagname>This is content</tagname>
 ```
 
-We can even put tags inside of other tags, just make sure you're closing the elements in the right order
+We can even put tags inside of other tags, just make sure you're closing the tags in the right order.
+
 ```html
-<tagname>This is content <tagname2>This is other content</tagname></tagname>
+<tagname>This is content <tagname2>This is other content</tagname2></tagname>
 ```
 
 Of course though, there are no tags called `<tagname>`, there are more descriptive tags we can use to build our webpages. Some available tags are:
 
 - `<p>` - Paragraphs used to 'wrap' text.
-- `<header>` - The header tag is used to show the header of a page or section
-- `<section>` - A section of the page
-- `<article>` _ An article, or group of content on the page
-- `<h1>`, `<h2>`, `<h3>` - Content headings. Can be used to create importance of content throughout page.
+- `<header>` - The header tag is used to show the header of a page or section.
+- `<section>` - A section of the page.
+- `<article>` _ An article, or group of content on the page.
+- `<h1>`, `<h2>`, `<h3>` - Content headings. Can be used to create importance of content throughout page. An `h1` is the more important heading on the page.
 - `<footer>` - The footer tag can be used to group content together that completes the page or the section.
 
 ## Links
-The internet is made up of files that we can load into our browsers. To see these pages we need to use 'links' to see them. Within our web pages, we can make text or a button 'link' to another page so we can help people navigate the internet.
```

Comment:
AP Styleguide is lowercasing it now - https://twitter.com/APStylebook/status/716384777406922753